### PR TITLE
Add missing fields to order model

### LINKED ISSUE #738: Missing fields from order model due to wrong documentation on API
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**

Fields are missing on the order model and also, order ID is not being used while displaying orders, because of missing documentation on API. 
**Expected behavior:**

All the fields should be there according to the response received

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

- Add missing fields to order model
- Add order Id in the oder Id field

Fixes #738

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-739-open-event-frontend-fossasia.surge.sh

File: app/models/order.js [hunk 1]
```diff
@@ -11,7 +14,18 @@ export default ModelBase.extend({
...
   paymentMode    : attr('string'),
   status         : attr('string'),
+  transactionId  : attr('string'),
+  expMonth       : attr('string'),
+  expYear        : attr('string'),
+  identifier     : attr('string'),
+  brand          : attr('string'),
+  last4          : attr('string'),
+  paidVia        : attr('string'),
+  completedAt    : attr('moment'),
```
Comment (geekyd) [inline app/models/order.js]: Mark this `readonly`

File: app/models/order.js [hunk 2]
```diff
@@ -11,7 +14,18 @@ export default ModelBase.extend({
   zipcode        : attr('string'),
   paymentMode    : attr('string'),
   status         : attr('string'),
+  transactionId  : attr('string'),
+  expMonth       : attr('string'),
+  expYear        : attr('string'),
+  identifier     : attr('string'),
```
Comment (geekyd) [inline app/models/order.js]: I think this should also be readonly @niranjan94 ?

Comment (CosmicCoder96) [inline app/models/order.js]: yup

**Comment by CosmicCoder96** [comment]
@geekyd Done !

**Comment by niranjan94** [comment]
Deployment link: https://pr-739-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-739-open-event-frontend-fossasia.surge.sh

**Pull request merged**